src/main/java/uk/gov/ons/census/fwmt/jobservice/rabbit/RmReceiver.java

```diff
@@ -92,14 +94,17 @@ public void receiveCreateMessage(FwmtActionInstruction rmRequest, @Header("times
       default:
         break; //TODO THROW ROUTUNG FAILURE
       }
+    } catch (RestClientException e) {
+      log.error("- Create Message - Error sending message - {}  error - {} ", rmRequest, e.getMessage(), e);
+      transientExceptionHandler.handleMessage(message);
     } catch (Exception e) {
       log.error("- Create Message - Error sending message - {}  error - {} ", rmRequest, e.getMessage(), e);
-      gatewayRabbitTemplate.convertAndSend("GW.Error.Exchange", "gw.receiver.error", message);
+      gatewayRabbitTemplate.convertAndSend("GW.Error.Exchange", "gw.permanent.error", message);
```

Comment:
I don't work on your team, but am following this with interest. I would say that RM regularly self-heals from a number of glitches (network outages, zone outages, Rabbit node problems, DB outages) and we don't ever consider anything to be a "permanent" error until a person has reviewed it... often times we fix some underlying infrastructure problem (or it's fixed by Google) and then our system self-heals.

Just my unsolicited 2 cents.

Reply:
@NickGrantONS feel free to being nosey